.devcontainer/Dockerfile

```diff
@@ -42,4 +42,8 @@ RUN sh -c "$(wget -O- https://github.com/deluan/zsh-in-docker/releases/download/
 # Terraform
 RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - && \
     apt-add-repository "deb [arch=$(dpkg --print-architecture)] https://apt.releases.hashicorp.com $(lsb_release -cs) main" && \
-    apt-get -y install terraform
\ No newline at end of file
+    apt-get -y install terraform
+
+# Rustc and Cargo Install
```

Comment:
Maybe add a link to the docs where this shell script is recommended? (other dev-tools are installed along-side Rustc and Cargo)